client/collector.go

```diff
@@ -78,16 +68,11 @@ func (c *Client) GetCollector(appID string, specID string) (*collectors.AppColle
 
 }
 
-func (c *Client) PromoteCollector(appID string, specID string, channelIDs ...string) error {
-
-	appType, err := c.GetAppType(appID)
-	if err != nil {
-		return err
-	}
+func (c *Client) PromoteCollector(appID string, appType string, specID string, channelIDs ...string) error {
 
 	if appType == "platform" {
 		return c.PlatformClient.PromoteCollector(appID, specID, channelIDs...)
-	} else {
+	} else if appType == "ship" {
 		return c.ShipClient.PromoteCollector(appID, specID, channelIDs...)
 	}
```

Comment:
Is it intentional that we're not specifically explaining that Kots apps collectors are handled differently, as we do with the CreateCollector call above?

Reply:
My thought was that to call this you actual need to have created a spec and have a specID...which would be impossible if it was a kots app.

Reply:
*actually*